internal/services/monitor/monitor_action_group_data_source_test.go

```diff
@@ -90,9 +90,9 @@ func TestAccDataSourceMonitorActionGroup_complete(t *testing.T) {
 				check.That(data.ResourceName).Key("email_receiver.1.email_address").HasValue("devops@contoso.com"),
 				check.That(data.ResourceName).Key("email_receiver.1.use_common_alert_schema").HasValue("false"),
 				check.That(data.ResourceName).Key("itsm_receiver.#").HasValue("1"),
-				check.That(data.ResourceName).Key("itsm_receiver.0.workspace_id").HasValue("6eee3a18-aac3-40e4-b98e-1f309f329816"),
+				check.That(data.ResourceName).Key("itsm_receiver.0.workspace_id").HasValue("5def922a-3ed4-49c1-b9fd-05ec533819a3|55dfd1f8-7e59-4f89-bf56-4c82f5ace23c"),
```

Comment:
how come this changes to two guids?

Reply:
This should be included in PR https://github.com/hashicorp/terraform-provider-azurerm/pull/15422/files#diff-ce1deef2394f5c8e69678679decfbdbb0bd48b5502e39b25f1361edd28cead0bR500